exercises/hamming/hamming.spec.js

```diff
@@ -1,46 +1,47 @@
 import { compute } from './hamming';
+var strands = require('./strands.json')
```

Comment:
use const instead of var , if a variable doesnt change its value then its the best choice